src/NzbDrone.Core/Indexers/RssSyncService.cs

```diff
@@ -60,15 +67,70 @@ private ProcessedDecisions Sync()
             return processed;
         }
 
+        private IEnumerable<ReleaseInfo> FilterByIndexer(IEnumerable<ReleaseInfo> releases)
+        {
+            var indexerBackOff = new Dictionary<int, bool>();
+
+            foreach (var release in releases)
+            {
+                bool ignore;
+                if (!indexerBackOff.TryGetValue(release.IndexerId, out ignore))
```

Comment:
you should use `Cached`instead of doing your own cache. that's exactly why it's there.